beps/0005-split-backend-discovery/README.md

```diff
@@ -227,22 +296,20 @@ While we could attach the existing information to the `PluginMetadataService`, w
 
 ```ts
 interface InstanceMetadataService {
-  listFeatures: () => BackendFeature[];
-  // or
-  listFeatures: () => string[]; // list of pluginIds/moduleIds.
+  getInstalledFeatures: () => BackstageFeature[];
```

Comment:
Who will consume this service? Was the idea that this would be the local `DiscoveryService` implementation, in order to know what to route locally? If so, I'm not even completely sure about that - you generally want to reach outside your own node anyway (almost to the point of avoiding your own node), but then maybe you want to prioritize nodes that are in the same region as your own for latency and egress cost reasons etc. Kinda wondering if we could even skip this service until we know more, for example in terms of "do we need to have a reactive interface that tells me when things change", and "do we want technical info in this data, like package metadata and where it's sourced from", and "in the dynamic plugin world, do we want to list POTENTIAL installable things here as well as actually activated things" etc etc.

Reply:
Yup, the local `DiscoveryService` implementation, so it can route this locally. It saves an extra request by having requests locally route to the same instance, instead of having to request the list from a/the gateway node. I'm open to skipping this for now. 

Definitely a lot of cool things we can do with this :), reactivity + dynamic plugins would be real nice next steps